src/Controls/src/Core/Platform/Android/BottomNavigationViewUtils.cs

```diff
@@ -39,6 +40,31 @@ internal static void UpdateEnabled(bool tabEnabled, IMenuItem menuItem)
 				menuItem.SetEnabled(tabEnabled);
 		}
 
+		internal static Task SetupMenuItem(
+			(string title, ImageSource icon, bool tabEnabled) item,
+			IMenu menu,
+			int index,
+			int currentIndex,
+			BottomNavigationView bottomView,
+			IMauiContext mauiContext,
+			out IMenuItem menuItem)
+		{
+			Task returnValue;
+			using (var title = new Java.Lang.String(item.title))
```

Comment:
```suggestion
			using var title = new Java.Lang.String(item.title);
```
And remove not needed indentation.